sw/host/opentitanlib/src/image/testdata/manifest_missing.hjson

```diff
@@ -0,0 +1,104 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Manifest for boot stage images stored in flash.
+ *
+ * OpenTitan secure boot, at a minimum, consists of three boot stages: ROM,
+ * ROM_EXT, and the first owner boot stage, e.g. BL0. ROM is stored in the
+ * read-only Mask ROM while remaining stages are stored in flash. This structure
+ * must be placed at the start of ROM_EXT and first owner boot stage images so
+ * that ROM and ROM_EXT can verify the integrity and authenticity of the next
+ * stage and configure peripherals as needed before handing over execution.
+ *
+ * Use of this struct for stages following the first owner boot stage is
+ * optional.
+ *
+ * Note: The definitions in
+ * sw/host/rom_ext_image_tools/signer/image/src/manifest.rs must be updated if
+ * this struct is modified. Please see the instructions in that file.
+ */
+{
```

Comment:
I ended up leaving out `signature`, `usage_constraints.device_id`, `usage_constraints.life_cycle_state`, `timestamp`, and `binding_value` just to include at least one instance of each type in the struct definition.

For `usage_constraints` serde actually will complain if the field is missing. We can change this if you think the flexibility is useful.

Reply:
Matter of fact I was mistaken, since `ManifestUsageConstraintsDef` implements `Default` and gets marked as `#[serde(default)]` in the macro it is valid to have it missing just like the other fields. I've updated the test file to demonstrate this.